src/utils/Utils.ts

```diff
@@ -89,17 +89,43 @@ export default class Utils {
     return threePhases;
   }
 
+  public static getUsedPhasesInTransactionInProgress(chargingStation: ChargingStation, transaction: Transaction): CSPhasesUsed {
+    const currentType = Utils.getChargingStationCurrentType(chargingStation, null, transaction.connectorId);
+    if (currentType === CurrentType.AC && transaction.currentInstantAmps > 0) {
```

Comment:
Double check L1/2/3 also?
Helper method in Utils to check the 4 of them?
Consumption in W.h is always provided and if non null and you don't have Amps info, default on 3 phases.

Reply:
I implemented a method, but left out the check on the W.h, because I found metervalues in the logs, where instant amps were given, but W.h were 0 anyway.